libraries/standard/common/include/iot_taskpool.h

```diff
@@ -83,8 +83,8 @@
  *
  * This function should be called once by the application to initialize the one single instance of the system task pool.
  * An application should initialize the system task pool early in the boot sequence, before initializing any other library
- * that uses the system task pool. such as MQTT, Shadow, Defernder, etc.. An application should also initialize the system
- * task pool before posting any jobs. Early initialization it typically easy to accomplish by creating the system task pool
+ * that uses the system task pool. such as MQTT, Shadow, Defernder, etc. An application should also initialize the system
```

Comment:
More minor spelling and grammar: `the system task pool.` -> `the system task pool,` or just `it,`
And `Defernder` -> `Defender`

Reply:
Thanks